stage0/common.go

```diff
@@ -164,6 +164,18 @@ func generateRuntimeApp(appRunConfig *apps.App, am *schema.ImageManifest, podMou
 		ra.App.SupplementaryGIDs = appRunConfig.SupplementaryGIDs
 	}
 
+	for k, v := range appRunConfig.Annotations {
+		if _, ok := ra.Annotations.Get(k); ok {
+			continue
+		}
+		kAci, err := types.NewACIdentifier(k)
+		if err != nil {
+			return ra, err
```

Comment:
Can we do

```go
return errwrap.Wrap(fmt.Errorf("error parsing annotation key %q", k), err)
```

I'd like to avoid confusing errors like these:

```
> sudo -E rkt --debug app add  64a8c4bf  kinvolk.io/aci/busybox --annotation=FOO=BAR --name=buzyboxz
stage0: locking pod manifest
stage0: Loading image sha512-140375b2a2bd836559a7c978f36762b75b80a7665e5d922db055d1792d6a4182
stage0: Writing image manifest
add: error adding app to pod
  └─ACIdentifier must contain only lower case alphanumeric characters plus "-._~/"
```